net-tools/netwatch/src/interfaces.rs

```diff
@@ -262,8 +261,7 @@ impl State {
     /// free to add parameters or different alternative constructors.
     ///
     /// [`ProbePlan`]: crate::netcheck::reportgen::probes::ProbePlan
```

Comment:
this reference is not available anymore

Reply:
solved in [e9959e5](https://github.com/n0-computer/iroh/pull/2855/commits/e9959e5eee8b20e6d6ad279b1682470ab9f9dccf)